### PR TITLE
Release/v0.2.0

### PR DESCRIPTION
**Pull request by AP6YC**
This PR folds in release v0.2.0. This new subversion changes the internals with privatization and adds more CVIs.

### CODE REVIEW

File: src/cvi/modules/CH.py [hunk 1]
```diff
@@ -36,21 +36,21 @@ def __init__(self):
...
         """
         Calinski-Harabasz (CH) setup routine.
         """
 
         # Run the generic setup routine
-        super().setup(sample)
+        super()._setup(sample)
 
         # CH-specific setup
         self.SEP = np.zeros([self.dim])
```
Comment (NiklasMelton) [inline src/cvi/modules/CH.py:49]: make these private

File: src/cvi/modules/GD43.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    def __init__(self):
+        """
+        Generalized Dunn's Index 43 (GD43) initialization routine.
+        """
+
+        # Run the base initialization
+        super().__init__()
+
+        # CH-specific initialization
+        self.mu = np.zeros([0])     # dim
```
Comment (NiklasMelton) [inline src/cvi/modules/GD43.py:35]: make these private

File: src/cvi/modules/GD43.py [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+                (1 - 1/n_samples_new) * self.mu
+                + (1/n_samples_new) * sample
+            )
+
+        # IF NEW CLUSTER LABEL
+        # Correct for python 0-indexing
+        if i_label > self.n_clusters - 1:
+            n_new = 1
+            v_new = sample
+            CP_new = 0.0
```
Comment (NiklasMelton) [inline src/cvi/modules/GD43.py:78]: make private

File: src/cvi/modules/GD43.py [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+                    )
+                D_new[i_label, :] = d_column_new
+                D_new[:, i_label] = d_column_new
+
+            # Update 1-D parameters with list appends
+            self.n_clusters += 1
+            self.n.append(n_new)
+            self.CP.append(CP_new)
+
+            # Update 2-D parameters with numpy vstacks
```
Comment (NiklasMelton) [inline src/cvi/modules/GD43.py:98]: make private

File: src/cvi/modules/GD43.py [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+            d_column_new = np.zeros(self.n_clusters)
+            for jx in range(self.n_clusters):
+                # Skip the current i_label index
+                if jx == i_label:
+                    continue
+                d_column_new[jx] = (
+                    np.sqrt(np.sum((v_new - self.v[jx, :]) ** 2))
+                )
+
+            # Update parameters
```
Comment (NiklasMelton) [inline src/cvi/modules/GD43.py:132]: should all be private

File: src/cvi/modules/GD43.py [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+    def _param_batch(self, data: np.ndarray, labels: np.ndarray):
+        """
+        Batch parameter update for the Generalized Dunn's Index 43 (GD43) CVI.
+        """
+
+        # Setup the CVI for batch mode
+        super()._setup_batch(data)
+
+        # Take the average across all samples, but cast to 1-D vector
+        self.mu = np.mean(data, axis=0)
```
Comment (NiklasMelton) [inline src/cvi/modules/GD43.py:153]: should all be private

File: src/cvi/modules/GD53.py [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+
+    def __init__(self):
+        """
+        Generalized Dunn's Index 53 (GD53) initialization routine.
+        """
+
+        # Run the base initialization
+        super().__init__()
+
+        # CH-specific initialization
```
Comment (NiklasMelton) [inline src/cvi/modules/GD53.py:34]: make private

**Review by NiklasMelton — CHANGES_REQUESTED** [review]
Looks good but make all the internal variables private

File: src/cvi/modules/CH.py (hunk 1 above)
Comment (AP6YC) [reply src/cvi/modules/CH.py:49]: Same as #51

**Pull request merged**